src/app/src/Map.js

```diff
@@ -1,19 +1,61 @@
 import React, { Component } from 'react';
-import ReactMapGL, { NavigationControl } from 'react-map-gl';
+import ReactMapGL, {
+    NavigationControl,
+    Marker,
+    FlyToInterpolator,
+} from 'react-map-gl';
 import 'mapbox-gl/dist/mapbox-gl.css';
 import { connect } from 'react-redux';
 
-import onChangeViewport from './map.actions';
+import { onChangeViewport } from './map.actions';
 import './Map.scss';
+import SensorMarker from './SensorMarker';
+import BackToMapButton from './BackToMapButton';
+import sensors from './sensors.json';
+import { toggleBackToMapButton } from './map.actions';
 
 class GLMap extends Component {
+    constructor(props) {
+        super(props);
+
+        this.goToLocation = this.goToLocation.bind(this);
+        this.renderCityMarkers = this.renderCityMarkers.bind(this);
```

Comment:
Just occurred to me: CRA has the `transform-class-properties` babel transformer built in, which makes it possible *not* to use this `bind(this)` syntax if we want to use it.

The syntax would be like

```
goToLocation = (options) => {
    ...whatever
}
```

...and it would automatically bind to the class, so the whole constructor can go.

Reply:
(It also occurs to me that this could just be a function component at present, since it doesn't use any life cycle methods).

Reply:
👍